### ISSUE TITLE
Plain text email addresses in output aren't obfuscated

### ISSUE THREAD

**Issue by sebastian-marinescu** [Jako/emo#2]
After installing I immediately tested it, and it seems to not obfuscate the plain mails. 
From the test-resource the html-output looks like this:

```
<p>
  This is a test: test@mail.de 
</p>

<p>
  This is a test <span id="_emoaddrId3">
  <span class="emo_address">Bitte Javascript aktivieren!</span></span>
</p>

<p>
  This is a test <span id="_emoaddrId4">
  <span class="emo_address">Bitte Javascript aktivieren!</span></span>
</p>

<p>
  This is a test: user@host.com 
</p>
```

The two mailto-links in the middle are obfuscated correctly, but the plain text mails not.

**Comment by Jako** [comment]
Fixed with 081a8eb62028bf473e11eb20a6fa9e2f6c9826c2

**Comment by sebastian-marinescu** [comment]
Looks good, will test it when it's available in the Extras-Repository :+1:
